contributor-docs/adrs/adr-008-experimental-components.md

```diff
@@ -47,15 +47,15 @@ The approach that has served us well since Dec 2021 has been "drafts" (along wit
 
 1. npm package: "Experimental" components, as defined by the component lifecycle, should not be part of semantically versioned npm package for `@primer/react`.
 
-2. npm package: Each experimental component should be independently published and versioned with semver. Breaking changes should be tagged as major versions. Example: `@github/react-commentbox`
+2. npm package: Each experimental component should be independently published and versioned with semver. Breaking changes should be tagged as major versions. Example: `@github/experimental-react-commentbox`
 
-3. The code for experimental components should live in [github/github/modules/react-shared](https://github.com/github/github/tree/master/app/assets/modules/react-shared) and published as individual packages as suggested in point 2.
+3. The code for experimental components should live in a new repository, example: [github/react-experimental](https://github.com/github/react-experimental) and published as individual packages as suggested in point 2.
```

Comment:
Working on code which is part of two repos has always been a pain point for me personally. How can we make sure that when extracting code into its own repo we still have a smooth developer experience and the feeling that we are still in one repo?

Reply:
1. My assumption is [npm link](https://docs.npmjs.com/cli/v8/commands/npm-link) would make this feel like one repo / one application
2. canary builds will as well
3. You'll still need to merge the component repo PR to be able to publish a new version. The goal would be to make that super quick with self review checklist and automated review in CI

Reply:
I think NPM link can work, but there's often the "2 version of react" scenario, leading to a 

link react/react-dom from the monolith in separate repo
link component from separate repo in dotcom

which can get hairy. 

Not sure I have much to offer here, but it's worth considering whether we can improve that flow somehow (not sure we can, just thinking if we can, we should try to make it foolproof.  `npm link` is already finnicky at times